### PR TITLE
food-ketchup

### PR DESCRIPTION
**Pull request by Szeraax**

### CODE REVIEW

File: data/food/base/data.json [hunk 1]
```diff
@@ -68,6 +68,41 @@
             "fdc_id": "168936",
             "alias": [],
             "comment": "all purpose flour, equivalent to type 550"
+        },
+        "food-ketchup": {
+            "name": "Ketchup",
+            "plural_name": "Ketchup",
+            "supermarket_category": "category-baking",
```
Comment (rsch136) [inline data/food/base/data.json:75]: Should this really be in baking? I would have looked for ketchup in condiments (with mustard, mayo, tabasco) or sauces (with pesto, bolognese, vanilla sauce).

File: data/food/base/data.json [hunk 2]
```diff
@@ -68,6 +68,41 @@
...
+            "preferred_unit_metric": "unit-g",
+            "preferred_packaging_unit_metric": "unit-g",
```
Comment (rsch136) [inline data/food/base/data.json:77]: My recipes all use ml and not grams for ketchup. Interestingly, mayo is measured in grams. The ketchup in my fridge is labeled in ml. The mayo in ml as well as g.

Comment (Szeraax) [reply data/food/base/data.json:77]: ml updated. Thanks!

File: data/food/base/data.json (hunk 1 above)
Comment (Szeraax) [reply data/food/base/data.json:75]: This is a good question. have you taken a look at the supermarket categories? How many categories would you think are worth adding to this base setup? https://github.com/TandoorRecipes/open-tandoor-data/blob/main/data/category/base/data.json

Comment (rsch136) [reply data/food/base/data.json:75]: I'll make a pr for some additional categories

Comment (rsch136) [reply data/food/base/data.json:75]: See #8

**Comment by vabene1111** [comment]
added to https://app.tandoor.dev/open-data/

**Pull request closed without merging**